Author: José Manuel Santamaría Lema <panfaust@gmail.com>
Description: This patch reverts the following upstream commit: 
 https://projects.kde.org/projects/kde/kde-baseapps/konsole/repository/revisions/6167fc97
 This patch MUST be removed for 4.9.x since the lost functionality would be
 available as a new "Clone Tab" action.
--- a/src/MainWindow.cpp
+++ b/src/MainWindow.cpp
@@ -397,14 +397,12 @@ void MainWindow::openUrls(const QList<KUrl>& urls)
 
 void MainWindow::newTab()
 {
-    Profile::Ptr defaultProfile = SessionManager::instance()->defaultProfile();
-    emit newSessionRequest(defaultProfile , activeSessionDir() , _viewManager);
+    emit newSessionRequest( _defaultProfile , activeSessionDir() , _viewManager);
 }
 
 void MainWindow::newWindow()
 {
-    Profile::Ptr defaultProfile = SessionManager::instance()->defaultProfile();
-    emit newWindowRequest(defaultProfile , activeSessionDir());
+    emit newWindowRequest( _defaultProfile , activeSessionDir() );
 }
 
 bool MainWindow::queryClose()

